src/sample_input_1.txt

```diff
@@ -0,0 +1,7 @@
+#BeginStory
```

Comment:
Should we have a space after #? I think it might make this a bit cleaner.

Reply:
That could work, it doesn't make a difference as far as the parser in concerned.